src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Decentralized finance (defi)
+description: An overview of defi on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+image: ../../assets/use-cases/defi.png
+sidebarDepth: 2
+summaryPoints:
+  [
+    "A global, open alternative to the current financial system.",
+    "Probably the biggest use case for Ethereum right now.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+_Defi is an alternative to a financial system that many of us have grown to mistrust thanks to the cycles of mismanagement, crashes, and bail outs. It's an open global system that gives you more control and visibility over your money. It's exposure to global markets when your currency at home is failing. And there are no banker bonuses – it's largely owned by its users or maintained as a public good. And right now there's over 40 billion dollars worth of crypto in defi applications._
+
+## What's defi?
+
+Defi is a collective term for products that are making financial products and services accessible to anyone who can use Ethereum – anyone with an internet connection. With defi, there are no centralized authorities who can block payments or deny you access to anything.
+
+There's a booming crypto economy out there, where you can lend, borrow, long/short, earn interest, and more. Crypto-savvy Argentinians have used defi to escape crippling inflation. Companies have started streaming their employees their wages in real time. Some folks have even taken out and paid off loans worth millions of dollars without the need for any personal identification.
+
+## Defi vs traditional finance
+
+One of the best ways to see the potential of defi is to understand the problems that exist today.
+
+- Some people aren't granted access to set up a bank account or use financial services
+- Financial services can block you from getting paid
+- The hidden charge of financial services is your personal data
+- Governments can close down markets at will
+- Money transfers can take days due to internal human processes
+- There's a premium to financial services because intermediray institutions need their cut
+
+| Defi                        | Traditional finance                                      |
+| --------------------------- | -------------------------------------------------------- |
+| You control your funds      | Your funds are looked after by companies                 |
+| Transfers happen in minutes | Payments can take days due to manual processes           |
+| Pseudonymous                | Financial activity is tightly coupled with your identity |
+| Open to anyone              | You must apply to use financial services                 |
```

Comment:
I suggest an additional row:
Markets are open 24/7/365 ("always open"?) vs. markets that close on weekends & (arbitrary) holidays

Reply:
And no more "after hours" trading for some

Reply:
@wackerow can you elaborate